modules/openapi-generator/src/main/resources/python-pydantic-v1/api_client.mustache

```diff
@@ -220,6 +220,12 @@ class ApiClient:
                                                      collection_formats)
             url += "?" + url_query
 
+        def extract_charset(content_type):
+            match = None
+            if content_type is not None:
+                match = re.search(r"charset=([a-zA-Z\-\d]+)[\s;]?", content_type)
+            return match.group(1) if match else "utf-8"
+
```

Comment:
thanks for the PR.

instead of using this, what about using the function that usually comes with the Python HTTP libraries instead, e.g. https://stackoverflow.com/questions/14592762/a-good-way-to-get-the-charset-encoding-of-an-http-response-in-python ?

(so that we don't need to reinvent the wheel and you may need to add another method in rest.py to return the charset)